nx-libs.spec

```diff
@@ -804,23 +771,6 @@ ln -s -f ../../../../%{_lib}/libXext.so.6 %{buildroot}%{_libdir}/nx/X11/Xinerama
 %defattr(-,root,root)
 %{_libdir}/libNX_Xau.so.6*
 
-%files -n libNX_Xcomposite1
-%defattr(-,root,root)
-%{_libdir}/libNX_Xcomposite.so.1*
-
-%files -n libNX_Xdamage1
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdamage.so.1*
-
-%files -n libNX_Xdmcp-devel
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdmcp.so
-%{_includedir}/nx/X11/Xdmcp.h
-
-%files -n libNX_Xdmcp6
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdmcp.so.6*
-
```

Comment:
Agreeing on the SUSE approach.

Add libx<lib><sonumber> manually under Depends: is not needed in Debian/Ubuntu and not correct. The dependencies are all gathered automatically under Depends: in bin:package nxagent via automatic library detection. We are not renaming those packages, we are switching to a different library via Build-Depends:.